src/trace_read.cc

```diff
@@ -71,88 +69,81 @@ POSSIBILITY OF SUCH DAMAGE.
 
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
-
-#define DEBUG(args...)   _DEBUG(*KNOB(KNOB_DEBUG_TRACE_READ), ## args)
-
+#define DEBUG(args...) _DEBUG(*KNOB(KNOB_DEBUG_TRACE_READ), ##args)
 
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
-
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
-
-trace_info_s::trace_info_s() { }
-trace_info_s::~trace_info_s() { }
-
+trace_info_s::trace_info_s() {}
+trace_info_s::~trace_info_s() {}
 
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
-
 reg_info_s::reg_info_s()
 {
-  m_reg  = 0;
+  m_reg = 0;
   m_type = INT_REG;
-  m_id   = 0;
+  m_id = 0;
 }
 
-
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
-
 trace_uop_s::trace_uop_s()
 {
-  m_opcode          = 0;
-  m_op_type         = UOP_INV;
-  m_mem_type        = NOT_MEM;
-  m_cf_type         = NOT_CF;
-  m_bar_type        = NOT_BAR;
-  m_num_dest_regs   = 0;
-  m_num_src_regs    = 0;
-  m_mem_size        = 0;
-  m_inst_size       = 0;
-  m_addr            = 0;
-  m_va              = 0;
-  m_actual_taken    = false;
-  m_target          = 0;
-  m_npc             = 0;
-  m_pin_2nd_mem     = false;
-  m_info            = NULL;
-  m_rep_uop_num     = 0;
-  m_eom             = false;
-  m_alu_uop         = false;
-  m_active_mask     = 0;
-  m_taken_mask      = 0;
+  m_opcode = 0;
+  m_op_type = UOP_INV;
+  m_mem_type = NOT_MEM;
+  m_cf_type = NOT_CF;
+  m_bar_type = NOT_BAR;
+  m_num_dest_regs = 0;
+  m_num_src_regs = 0;
+  m_mem_size = 0;
+  m_inst_size = 0;
+  m_addr = 0;
+  m_va = 0;
+  m_actual_taken = false;
+  m_target = 0;
+  m_npc = 0;
+  m_pin_2nd_mem = false;
+  m_info = NULL;
+  m_rep_uop_num = 0;
+  m_eom = false;
+  m_alu_uop = false;
+  m_active_mask = 0;
+  m_taken_mask = 0;
   m_reconverge_addr = 0;
-  m_mul_mem_uops    = false;
+  m_mul_mem_uops = false;
 
-  m_hmc_inst        =HMC_NONE;
+  m_hmc_inst = HMC_NONE;
 }
 
-
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
 #ifdef USING_QSIM
-trace_gen* trace_read_c::m_tg;
+trace_gen *trace_read_c::m_tg;
 #endif
 
 /**
  * Constructor
  */
-trace_read_c::trace_read_c(macsim_c* simBase, ofstream* dprint_output)
+trace_read_c::trace_read_c(macsim_c *simBase, ofstream *dprint_output)
 {
   m_simBase = simBase;
   m_dprint_count = 0;
   m_dprint_output = dprint_output;
 
 #ifdef USING_QSIM
   using namespace Qsim;
-  if (!m_tg) {
```

Comment:
i think this is against our format guideline

Reply:
@hyojongk What exactly seems to be the problem? Is it the use of the ! operator? or the naming convention? 
Thanks